atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -222,6 +222,32 @@ def get_remote_urls(self):
 
         return remote_sources_urls
 
+    def get_blacklisted_sources(self):
```

Comment:
Gotta start somewhere...

This should really be `denylisted`/`rejectlisted`/similar.

Same for other occurances.

Reply:
dunno we internally used 'blacklisted' so I am keeping it that way

Reply:
renamed